kernel/cClosure.ml

```diff
@@ -317,19 +319,19 @@ and fterm =
   | FConstruct of pconstructor
   | FApp of fconstr * fconstr array
   | FProj of Projection.t * fconstr
-  | FFix of fixpoint * fconstr subs
```

Comment:
Despite being more annoying to write, I suspect that expanding `subs` here to save an indirection would reduce the runtime overhead observed in mathcomp.